packages/braid-design-system/lib/components/Text/TextContext.ts

```diff
@@ -1,4 +1,4 @@
 import { createContext } from 'react';
-import { UseTextProps } from '../../hooks/typography';
+import type { TextStyleProps } from '../../css/typography';
 
-export const TextContext = createContext<UseTextProps | false>(false);
+export const TextContext = createContext<TextStyleProps | false>(false);
```

Comment:
My initial reaction was 🤨
After searching through the codebase I only found this instance where this pattern is used. Everywhere else it was

```ts
const SomethingContext = createContext<Something | null>(null);
```

Reply:
Interesting, I hear your point. We do use a boolean for context elsewhere (e.g. `HeadingContext`, `Modal`, etc.). But the way the code can be simplified with null coalescing probably justifies the change.